src/index.js

```diff
@@ -0,0 +1,171 @@
+import './styles.css';
+import { ShowElement, HideElement } from './visiable.js';
+import CheckIfCompleted from './completion.js';
+
+let tasks = [];
+
+const enterIcon = document.getElementById('enter-icon');
+const addField = document.getElementById('add-field');
+const cleanBtn = document.getElementById('clean-btn');
+const listContainer = document.getElementById('list-container');
+let ShowList;
+
+if (localStorage.getItem('to_do_list') !== null) {
+  tasks = JSON.parse(localStorage.getItem('to_do_list'));
+}
+
+const UpdateStorage = () => localStorage.setItem('to_do_list', JSON.stringify(tasks));
```

Comment:
I would recommend using `camelCase` approach for functions as it is a JavaScript best practice.